journal/updated/05-chapter-4.adoc

```diff
@@ -39,205 +39,205 @@ Bristol, Fourth month 30th, 1763.
 
 Though I am far from being lifted up,
 yet I esteem it a great favour that I have hitherto been preserved from sinking too low;
-therefore I hope you will continue easy about me; for though I am yours in one sense,
-yet as we are none of us our own, let us be willing to serve Him whose we are,
+therefore I hope you will continue easy about me. For though I am yours in one sense,
+yet none of us are our own; therefore let us be willing to serve Him to whom we belong,
 and be obedient children unto Him who in love visited us.
-Surely,
-He it is who sees fit to separate us outwardly--yet I am often near to you in spirit.
+Surely it is He who sees fit to separate us outwardly--yet I am often near to you in spirit.
 I have had many satisfactory opportunities in this city,
 and my friends seem to have unity with me, which is a strength to me.
-I cannot forget what I have met with in my painful pilgrimage;
-but have still hope that it has been, and will be of service to me,
+I cannot forget what I have met with in my painful pilgrimage,
+but still have hope that it has been and will be of service to me
 by making me more watchful and careful, for there is great need of watchfulness.
 
 Many are ill,
 and many die suddenly--how happy are all those who have made their peace with God!
-I think I may say, I know that peace, and beg that I may be preserved to the end;
-being assured,
-that all those who are '`faithful unto death,`' shall receive '`a crown of life.`'
+I think I may say, "`I know that peace,`" and beg that I may be preserved to the end,
```

Comment:
I don't think the quotes are necessary here. She's not really quoting anybody, or herself... just saying that she thinks she is able to say this in truth.